Rettelse PR (#62)
MrLongbottom/P9-Rapport: sections/Experiment.tex

\section{Experiment}\label{sec:experiment}

The goal of our experiment is to test two different aspects of quality for combinations of \gls{ir} methods.
We test the \gls{ir} methods' ability to find a specific document based a on query.
To do this we generate queries based on documents in the corpus, as described in \autoref{subsec:query_gen_doc}, and then test the \gls{ir} methods' ability to find the original document that each query was based upon.

We also test the \gls{ir} methods' ability understand the queries on a more abstract level and find documents are within the same topic as the query.
To do this, we generate queries based on topics from our \gls{lda} model, as described in \autoref{subsec:query_gen_top}, and test the \gls{ir} methods' ability to find all documents that are relevant to the original topic that the query was based upon.

We test combinations of the following \gls{ir} methods (these will be referred to as the baselines):
\begin{itemize}

> 	\item \acrfull{lm}
> 	\item \acrlong{lda}-\acrlong{ir} (\acrshot{lda}-\acrshort{ir})

Reviewer: ```suggestion
	\item \acrlong{lda}-\acrlong{ir} (\acrshot{lda}-\acrshort{ir})
	\item \acrfull{lm}
```

\section{Experiments}\label{sec:experiment}

The goal of our experiment is to test two different aspects of quality for combinations of \gls{ir} methods.
We test the \gls{ir} methods' ability to find a specific document based a on query.
To do this we generate queries based on documents in the corpus, as described in \autoref{subsec:query_gen_doc}, and then test the \gls{ir} methods' ability to find the original document that each query was based upon.

We also test the \gls{ir} methods' ability understand the queries on a more abstract level and find documents are within the same topic as the query.
To do this, we generate queries based on topics from our \gls{lda}, as described in \autoref{subsec:query_gen_top}, and test the \gls{ir} methods' ability to find all documents that are relevant to the original topic that the query was based upon.

We test combinations of the following \gls{ir} methods:
\begin{itemize}
	\item \acrfull{lm}

> 	\item \gls{lda}-\gls{ir}

Author: ```suggestion
	\item \acrlong{lda}-\acrlong{ir} (\gls{lda}-\gls{ir})
```

We use the term \gls{lda}-\gls{ir}, to refer to information retrieval made using \autoref{eq:lda-ir}.

\todo[inline]{describe bm25}

\todo[inline]{describe tf-idf}

Each baseline will be given a query and produce a document-value vector, based on that query, with one value for each document indicating the relevance of that document.
Our evaluation results are based on the ranking order of these values.
When multiple baselines are combined, their document-value vectors are normalized and then added or multiplied together element-wise.
However, in this experiment we will not only test the ability to retrieve a specific document based on a query related to the document, but also the ability to retrieve relevant documents within a specific topic, generated by our topic model.

For each baseline, we evaluate the quality of its \gls{ir} for both queries generated based on documents (document queries) and queries generated based on topics (topic queries).

> For topic queries, all documents with values higher than the mean of the topic's vector in the document-topic matrix $\theta_t$ are considered correct retrievals.

Author: ```suggestion
For topic queries, all documents with values higher than the mean of the topic's vector in the document-topic matrix $\theta_t$ are considered ground truth positives.
```

		\gls{lm} + \gls{pr} & 0.170 & 0.153 & 0.283 & 0.256 & 0.130 & 0.132 & 0.130 & 0.131 \\
		\gls{lm} * \gls{pr} & 0.163 & 0.138 & 0.259 & 0.236 & 0.130 & 0.133 & 0.129 & 0.130 \\
		\gls{lm} + \gls{lda}-\gls{ir} + \gls{pr} & 0.0499 & 0.0214 & 0.0601 & 0.120 & 0.148 & 0.164 & 0.146 & 0.147 \\
		\gls{lm} * \gls{lda}-\gls{ir} * \gls{pr} & 0.0911 & 0.0459  & 0.157 & 0.170 & 0.150 & 0.175 & 0.155 & 0.166 \\
		\gls{bm25} + \gls{lda}-\gls{ir} & \textbf{0.276} & 0.524 & 0.588 & 0.365 & 0.155 & 0.184 & 0.168 & 0.176 \\
		\gls{bm25} * \gls{lda}-\gls{ir} & 0.139 & 0.270 & 0.412 & 0.365 & 0.154 & 0.159 & 0.156 & 0.162 \\
		\gls{bm25} + \gls{pr} & 0.269 & 0.656 & 0.866 & 0.902 & \textbf{0.192} & 0.193 & 0.175 & 0.183 \\
		\gls{bm25} * \gls{pr} & 0.267 & \textbf{0.663} & \textbf{0.884} & 0.904 & 0.155 & 0.159 & 0.155 & 0.161 \\
		\gls{bm25} + \gls{lda}-\gls{ir} + \gls{pr} & \textbf{0.276} & 0.525 & 0.589 & 0.366 & 0.162 & 0.192 & 0.176 & 0.184 \\
		\gls{bm25} * \gls{lda}-\gls{ir} * \gls{pr} & 0.150 & 0.266 & 0.446 & 0.381 & 0.155 & 0.159 & 0.156 & 0.163 \\
	\end{tabular}
	
	\label{tab:results}
\end{table*}

\begin{table*}[h]
	\centering

> 	\caption{Results table}

Author: tror vi har et issue med det?
